XRTK.SDK/Packages/com.xrtk.sdk/Runtime/Features/UX/Scripts/Controllers/Hands/BaseHandControllerVisualizer.cs

```diff
@@ -113,8 +113,8 @@ private void UpdateHandJointTransforms(HandData handData)
                 {
                     var jointTransform = GetOrCreateJointTransform(handJoint);
                     var jointPose = jointPoses[handJoint];
-                    jointTransform.localPosition = jointPose.Position;
-                    jointTransform.localRotation = jointPose.Rotation;
+                    jointTransform.localPosition = jointTransform.parent.InverseTransformPoint(jointPose.Position);
```

Comment:
Is this bc the center position is never updated?

Reply:
This is a temporary fix I made to see hands again. I am planning to solve this in a different place, probably converter, so the visualizer always gets the correct poses.